relay-sampling/src/evaluation.rs

```diff
@@ -24,13 +28,115 @@ fn pseudo_random_from_uuid(id: Uuid) -> f64 {
     generator.sample(dist)
 }
 
+/// Increments the reservoir count for a given rule in redis.
+///
+/// - INCR docs: [`https://redis.io/commands/incr/`]
+/// - If the counter doesn't exist in redis, a new one will be inserted.
+#[cfg(feature = "redis")]
+fn increment_redis_reservoir_count(
+    redis: &Arc<RedisPool>,
+    org_id: u64,
+    rule_id: RuleId,
+) -> anyhow::Result<i64> {
+    let mut command = relay_redis::redis::cmd("INCR");
+
+    let key = format!("bias:{}:{}", org_id, rule_id);
+    command.arg(key);
```

Comment:
I would've expected clippy to alert on this.

```suggestion
    command.arg(format!("bias:{org_id}:{rule_id}"));
```

Separately, should we rather call this something like `reservoir:`, to make it more specific to what this counter is for? It's not unlikely that there will be other biases in the future that track stuff in Redis.